test/index.js

```diff
@@ -362,12 +365,12 @@ const doTest = (formatversion) => {
         let idx = 0;
         data.forEach((test) => {
             idx++;
-            it('Should covert byte range. Test ' + idx,  () => {
+            it(`Should covert byte range. Test ${idx}`,  () => {
                 assert.deepEqual(utils.convertByteClassToUnicodeClass(test[0]), test[1]);
             });
         });
 
-       /* it('Should fetch domains', () => {
+        /* it('Should fetch domains', () => {
```

Comment:
Oh, that's my fault... I've commented it out for testing and completely forgot.. Could you uncomment it as a part of this PR? It seems like a good piece for this commit message

Reply:
Done.